MonoGame.Framework/Graphics/GraphicsAdapter.cs

```diff
@@ -107,6 +111,12 @@ public DisplayMode CurrentDisplayMode
                     graphics.ReleaseHdc(dc);
                     return new DisplayMode(width, height, SurfaceFormat.Color);
                 }
+#elif WINDOWS_UAP
+                var visibleBounds = ApplicationView.GetForCurrentView().VisibleBounds;
+                var rawPixelsPerViewPixel = DisplayInformation.GetForCurrentView().RawPixelsPerViewPixel;
+                return new DisplayMode((int)(visibleBounds.Width * rawPixelsPerViewPixel + 0.5f),
+                                       (int)(visibleBounds.Height * rawPixelsPerViewPixel + 0.5f),
+                                       SurfaceFormat.Color);
 #else

```

Comment:
This works *if* the app is running full-screen, but that's not guaranteed or a requirement for a Win10 UWP.  Are you forcing to full-screen with TryEnterFullScreenMode or some such?  I just took your code and ran it against an app that starts in a window and I simply get the resolution of the window, not the desktop.